authmodules/TYPO3/privacyidea/ext_conf_template.txt

```diff
@@ -11,7 +11,7 @@ privacyIDEAURL = https://localhost
 privacyIDEACertCheck = 0
 
 # cat=Server/Optional; type=string; label=privacyIDEA realm
-privacyIDEARealm = ""
+privacyIDEARealm = defrealm
```

Comment:
It is not sure, if you have a realm called "defrealm". 
Please remove this again.
If the realm is empty (like it is configured originall), the default realm is used.
An empty realm always means default realm.